jobs/ingestor_archiver/spec

```diff
@@ -27,6 +27,18 @@ properties:
   logstash_ingestor.debug:
     description: Debug level logging
     default: false
+  logstash_ingestor.outputs:
+    description: |
+      A list of output plugins, with a hash of options for each of them. Please refer to example below.
+    example:
+      outputs:
```

Comment:
Traditionally-without-knowing-the-actual-tradition, the value of `example` should be whatever the value of the property is - same format as `defaults` - so I don't think the mid-level `outputs` key should be included.